OrbitGl/CaptureWindow.cpp

```diff
@@ -81,9 +81,10 @@ void CaptureWindow::MouseMoved(int x, int y, bool left, bool /*right*/, bool /*m
     time_graph_.GetWorldMinMax(world_min, world_max);
 
     auto width = static_cast<float>(GetWidth());
+    auto height = static_cast<float>(GetHeight());
 
     world_top_left_x_ = world_click_x_ - static_cast<float>(x) / width * world_width_;
-    world_top_left_y_ = world_click_y_ + static_cast<float>(y) / width * world_height_;
+    world_top_left_y_ = world_click_y_ + static_cast<float>(y) / height * world_height_;
```

Comment:
Actually, as there is now only one use of width/height, I would get rid of the variable again, and inline the call+cast.
https://google.github.io/styleguide/cppguide.html#Local_Variables also says that the variable should be as close as possible to the usage. So if you decide to keep it as a variable, you would need to move that variable directly to the line.

Reply:
Perfect, done!